lib/chromic_pdf.ex

```diff
@@ -181,6 +179,28 @@ defmodule ChromicPDF do
         ]
       end
 
+  ### Multiple session pools
+
+  ChromicPDF supports running multiple named session pools to allow varying session configuration.
+  For example, this makes it possible to one pool that is not allowed to execute JavaScript while
```

Comment:
I think there is a verb missing. Maybe "have"?

```suggestion
  For example, this makes it possible to have one pool that is not allowed to execute JavaScript while
```